app/src/main/java/com/folen/androidshowreel/activity/FragmentActivity.java

```diff
@@ -0,0 +1,86 @@
+package com.folen.androidshowreel.activity;
+
+import android.content.Context;
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentTransaction;
+import android.support.v7.widget.Toolbar;
+import android.view.Menu;
+import android.view.MenuItem;
+
+import com.folen.androidshowreel.R;
+import com.folen.androidshowreel.base.BaseActivity;
+import com.folen.androidshowreel.util.FirstFragment;
+import com.folen.androidshowreel.util.SecondFragment;
+
+public class FragmentActivity extends BaseActivity {
+
+    private Toolbar mToolbar;
+    private Fragment fragment;
+
+    @Override
+    protected void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_fragment);
+
+        setupToolbar();
+    }
+
+    private void setupToolbar() {
+        mToolbar = findViewById(R.id.fragment_toolbar);
+        setSupportActionBar(mToolbar);
+        mToolbar.setTitle("Fragments");
+        getSupportActionBar().setDisplayHomeAsUpEnabled(true);
+    }
+
+    @Override
+    public boolean onCreateOptionsMenu(Menu menu) {
+        getMenuInflater().inflate(R.menu.toolbar_menu, menu);
```

Comment:
ta metoda inflate nie zwraca przypadkiem boolean i nie powinnismy jej tu zwrocic? jesli nie to wydaje mi sie ze super.onCreateOptionsMenu bardziej sie tutaj  nadaje

Reply:
prosze wytłumacz, zostawiłem to bez zmian